features/account/registering.feature

```diff
@@ -0,0 +1,33 @@
+@customer_account
+Feature: Account registration
+    In order to order products
+    As a Visitor
+    I need to be able to create an account in the store
+
+    Background:
+        Given the store operates on a single channel in "France"
+
+    @ui
+    Scenario: Registering a new account with minimum information
+        Given I want to register a new account
+        When I specify first name with "Saul"
```

Comment:
I am pretty sure ``specify with`` is incorrect, how about ``specify as``? I'm still not 100% sure it is correct, but at least it is consistent with other scenarios and we can fix it easily all together if needed.